keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/SubmittedResource.kt

```diff
@@ -38,7 +38,7 @@ data class SubmittedResource<T : ResourceSpec>(
   val spec: T
 )
 
-val <T : ResourceSpec> SubmittedResource<T>.id: String
+val SubmittedResource<*>.id: String
```

Comment:
just curious, why the loosening of types here?

Reply:
Because the spec type doesn't matter to get the `id`. But this was a side-change, will undo.